templates/CRM/CloseAccountingPeriod/Form/PriorFinancialPeriod.tpl

```diff
@@ -38,7 +38,9 @@
 CRM.$(function($) {
 
   function checkPeriod() {
-    var speriod = $('#prior_financial_period').val();
+    var m = $('#prior_financial_period_M').val();
+    var y = $('#prior_financial_period_Y').val();
+    var speriod = m + '-' + y;
```

Comment:
@JoeMurray this again is not for viewing purposes. The date is only used to perform a backend validation check.